src/pages/swap/sections/BankDetails.tsx

```diff
@@ -16,11 +16,16 @@ export const BankDetails: FC<BankDetailsProps> = ({ registerBankAccount, registe
     <p className="my-5 font-thin text-center text-gray-400">Which bank account should we send the funds to?</p>
     <LabeledInput
       label="PIX ID"
```

Comment:
This whole component is not even used anymore. We should be able to remove the whole file.

Reply:
What is this component about?